.npmrc

```diff
@@ -1,2 +0,0 @@
-registry="https://registry.yarnpkg.com"
-strict-ssl=false
```

Comment:
we previously added this file to account for some package management behaviors. Does yarn 3 do away with the reliance on npm configuration in some cases?

Reply:
What scenarios is this needed for? In the case for yarn, it doesn't read `.npmrc` files any more. https://yarnpkg.com/getting-started/migration#dont-use-npmrc-files

Reply:
I believe `yarn info` relied on it among other commands we can into. This from the docs:

> _"Yarn doesn't use the configuration from your .npmrc files anymore"_

🎉